README.md

```diff
@@ -1,247 +1,8 @@
 # Apollo MCP Server
```

Comment:
@pubmodmatt would you mind adding in the Apollo graphic/header we use in other repos? E.g. https://github.com/apollographql/apollo-client (just the image itself at the top to keep brand synergy):

![CleanShot 2025-05-14 at 19 02 55@2x](https://github.com/user-attachments/assets/26c2b0e4-bf67-4789-bc7c-b124c2fd7c20)